logviewer/api/transport_linux.go

```diff
@@ -0,0 +1,81 @@
+// build +linux
+
+package api
+
+import (
+	"context"
+	"fmt"
+	"golang.org/x/sys/unix"
+	"net"
+	"os"
+	"runtime"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+)
+
+const titusInits = "/var/lib/titus-inits"
+
+// nsDialer is a net.Dialer that does a Setns() into a container's network namespace before doing a connect
+type nsDialer struct {
```

Comment:
Can you extract this into the utility package? This is a pattern that will be required, for example, by metadata service when running on kubelet.

Reply:
In person conversation: we agreed to leave it as-is, and @alekar will factor it out as part of the metadataservice CNI work.